File: 1758-minimum-changes-to-make-alternating-binary-string/1758-minimum-changes-to-make-alternating-binary-string.cpp
class Solution {
public:
    int minOperations(string s) {
        int res1 = 0;
        int res2 = 0;
        for(int i = 0; i < s.length(); i++)
        {
            if((s[i] == '1' && i % 2 == 0) || (s[i] == '0' && i % 2 == 1))
            {
                res1++;
            }
            else
            {
                res2++;
            }
        }
        return min(res1, res2);
    }
};

File: 1758-minimum-changes-to-make-alternating-binary-string/1758-minimum-changes-to-make-alternating-binary-string.py
class Solution:
    def minOperations(self, s: str) -> int:
        res1 = 0
        res2 = 0
        for i, ch in enumerate(s):
            if (ch == '1' and i % 2 == 0) or (ch == '0' and i % 2 == 1):
                res1 += 1
            else:
                res2 += 1
        return min(res1, res2)